.drone.yml

```diff
@@ -40,5 +43,6 @@ steps:
 trigger:
   event:
     - push
+    - tag
```

Comment:
Let's remove this trigger completely, e.g. we don't need to bump the tag on tag events

Reply:
Do you mean to remove push trigger?